src/client/features/interactions/cy/index.js

```diff
@@ -68,13 +68,41 @@ let bindEvents = cy => {
   cy.on('layoutstart', () => hideTooltips());
   cy.on('hide-type', () => hideTooltips());
   cy.on('slider-change', () => hideTooltips());
+
+  let nodeHoverMouseOver = _.debounce(evt => {
+    let node = evt.target;
+    let elesToHighlight = cy.collection();
+
+    //Create a list of the hovered node & its neighbourhood
+    node.neighborhood().nodes().union(node).forEach(node => {
```

Comment:
FYI `node.neighborhood().nodes().union(node)` is the same as `node.closedNeighborhood().nodes()`.